README.md

```diff
@@ -1,96 +1,129 @@
+Screenshot of findings shown in Xcode:
 
 ![Teamscale Findings in Xcode](config/xcode_2.png)
 
 ### VS Code
 
-Add a new task (`Terminal -> Configure Tasks`) and name it `Teamscale Precommit Analysis` or similar. VS Code will open a sample `tasks.json` for you to edit. Locate `config/teamscale-precommit-vscode-task.json` in this repo. Copy and paste the snippet and modify to your needs (e.g. `python` vs. `python3`).
+Add a new task (`Terminal > Configure Tasks > Create tasks.json file from template > Others`) and name it `Teamscale Pre-Commit Analysis`. VS Code will open a sample `tasks.json` for you to edit. Copy and paste [the example snippet from this repo](./config/teamscale-precommit-vscode-task.json).
 
 ### Vim
 
-Locate `config/teamscale.vim` and put in into `~/.vim/compiler`. Modify to your needs (e.g. `python` vs. `python3`).
-This should allow you to do `:compiler teamscale` and `:make %`. Then you should be able to use your usual workflow (e.g. `:cn`) to go through the findings.
+Copy [the teamscale-cli vim file](./config/teamscale.vim) to `~/.vim/compiler` and restart Vim.
+This should allow you to run `:compiler teamscale` and `:make %`. Then you should be able to use your usual workflow (e.g. `:cn`) to go through the findings.
+
+### QtCreator
 
-### QTCreator
-In order to use the precommit Analysis in QTCreator you need to add a new Kit (Teamscale does **not** have to be the default one)  
-Go to `Projects` -> `Manage Kits...` -> `Kits` -> `Add`  
+In order to use the pre-commit Analysis in QtCreator you need to add a new Kit (Teamscale does _not_ have to be the default Kit).
+Go to `Projects > Manage Kits... > Kits > Add`  
 
 ![New Teamscale Kit](config/qtcreator_1.png)
 
-The only thing important here to configure is that the compiler is gcc as this will be used to parse the output of the precommit analysis.
+The compiler must be set to gcc as this will be used to parse the console output of the tool.
 
-Next you need to add the actual run config. Go to `Projects` -> `Teamscale` -> `Build` and remove all default `Build` and `Clean Steps` (there should be a small x when hovering over them)  
-Then go to `Projects` -> `Teamscale` -> `Run`, and add a new `Run Configuration` (After doing so you can remove the default, as the Kit needs at least one). You can also rename the run config to, for example,
-**Precommit**
+Next you need to add the actual run config to each of your QtCreator projects.
+Go to `Projects > Teamscale > Build` and remove all default `Build` and `Clean Steps` (there should be a small `x` when hovering over them).
+Add a new build configuration:
 
-![Precommit](config/qtcreator_2.png)
+![Pre-Commit](config/qtcreator_2.png)
```

Comment:
whoops, I totally forgot that that was still open. Also didn't see your comment there about merging... I'll go over your changes there and make sure they're all incorporated here. My bad :-(

Reply:
Acutally, I prefer your dark screenshot because it also highlights how to run the multiple configs. Could you instead create a dark theme screenshot of the compiler settings? then I'll replace the first screenshot to have both in dark theme :-) @AnonymFx

Reply:
merged #14 and then merged master into this branch :-)

Reply:
I actually preferred replacing the dark theme screenshot as it was not optimal anyways in my opinion. Please check if that's fine for you as well :)

Reply:
It still shows the dark screenshot for me? https://github.com/cqse/teamscale-cli/blob/improve_docs/config/qtcreator_2.png

Reply:
Nope, that's actually the new one. Compare to https://github.com/cqse/teamscale-cli/blob/6eb45ba9113c66c94a56eaa699390a610a8251d0/config/qtcreator_2.png

Reply:
I guess the remaining dark elements are dark even in a light qt theme

Reply:
I see, weird :P works for me though